src/shared/hooks/useTransferOwnership.ts

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
```

Comment:
Please remove empty line

Reply:
it's the rule. it should be between internal and external imports